.nvmrc

```
@@ -0,0 +1 @@
+>=12.22
```

This is the version needed by `lint-staged`. I only know that `lint-staged@13` doesn't work on 12 because I'm upgrading the same package on the repos at work. 😆 
```suggestion
>=14.13.1
```

---

I don't understand what's the purpose to change it here? We're still supporting `v12` as mentioned above, so the contributor can also use `v12`?

---

The contributor can't use version 12 because `lint-staged@13` won't work. `lint-staged@12.5` will though.

---

https://github.com/okonet/lint-staged/blob/master/package.json

---

I've downgraded it in this commit. See [`8902edc` (#437)](https://github.com/prettier/prettier-eslint-cli/pull/437/commits/8902edc917a3e6d71cd0d0dda383ee99bce1c84a#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R55)